src/Lombiq.OSOCE.Web/Recipes/Lombiq.OSOCE.Tests.recipe.json

```
@@ -85,6 +85,114 @@
         ]
       }
     },
+    {
```

Could you include a comment on why this is necessary?

---

Sure. So for some reason, after updating the packages, these roles (that I added) were not created during testing by default thus, some tests failed because the homepage was not displaying (instead of it, it displayed a login page, because an anonymous user couldn't view the homepage), so I needed to add them. This same thing already happened to me in another project a while ago.

Should I add a comment about this to the code too?

---

Is this a problem during normal setup too or just during testing? 

> Should I add a comment about this to the code too?

Yes, please.

---

> Is this a problem during normal setup too or just during testing?
It's only present during testing.